### PR TITLE
Added blog post

### PR DESCRIPTION
**Pull request by kwennB**
Added the GSOD's team article.

**What kind of change does this PR introduce?**

Documentation

**Issue Number:**
-  Closes #___   https://github.com/json-schema-org/website/issues/816

**Does this PR introduce a breaking change?**

No

### CODE REVIEW

File: pages/blog/posts/GSOD-enhancing-json-schema-docs.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
+---
+title: 'Enhancing JSON Schema documentation: insights and progress from the Google Season of Docs 2024 project'
+date: '2024-09-03'
+tags:
+  - Google Season of Docs
+  - Documentation
+  - Community
+type: Documentation
+cover: img\posts\2024\GSOD-enhancing-json-schema-docs\JSON-website-cover.jpg
```
Comment (benjagm) [inline pages/blog/posts/GSOD-enhancing-json-schema-docs.md:9]: This is not the url of the image you added. In addition, please use PNG for better performance.

**Review by benjagm — CHANGES_REQUESTED** [review]
The article has too many sections with not much content what makes it difficult to follow the goal of the article. Uses less sections and less bullet points... and more storytelling.

Examples: 
https://hepsoftwarefoundation.org/gsdocs/blogs/2020/blog_ROOT_JohnYoon.html
https://forum.mautic.org/t/mautic-the-season-of-docs/18707


In addition, I am missing the personal opinion of the author, instead, this does not seem like a blog post... it seems something else. 

I will suggest simplifying the structure, to make it easy to follow.

Introduction
The project
The update
Closing thoughts

I will suggest to simplify the title: "Update from the Google Season of Docs 2024 Project"

**Review by gregsdennis — APPROVED** [review]
Vast improvement! Well done. 

🎉👏👏👏

File: pages/blog/posts/GSOD-docs-project-2024.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## The Update
+
+Since May, the team has been working hard to achieve these goals. We've conducted extensive research, mapped out [developer experience journeys](<(https://www.figma.com/board/TRjQUw33K93y8RlJMSRkJs/JSON-dev-journey?node-id=0-1&t=gYkdGtHk2sKrCQM0-0)>), implemented the [keywords page](https://json-schema.org/understanding-json-schema/keywords), and added on-page tables of content. We've also introduced [Next-Previous navigation](https://github.com/json-schema-org/website/pull/807) sections and restructured the [specification docs](https://github.com/json-schema-org/website/pull/823).
+
+What's pending? We're currently working on the [JSON Hyper-Schema](https://github.com/json-schema-org/website/issues/896) and [Migration pages](https://github.com/json-schema-org/website/issues/897) as part of the specification docs. Additionally, we're drafting style guides and creating [overview content buckets](https://github.com/json-schema-org/website/issues/803) to enhance the documentation.
```
Comment (benjagm) [inline pages/blog/posts/GSOD-docs-project-2024.md:33]: @valeriahhdez I think we need to add to this paragraph more details about what is pending from your side as this reflects mostly the pending work from Blessing's side.

File: pages/blog/posts/GSOD-docs-project-2024.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## The Project
+
+The main goal of JSON Schema's [documentation project](https://github.com/orgs/json-schema-org/discussions/687) is to create a scalable documentation architecture that is clear, consistent, and comprehensive. To achieve this through GSoD, we focused on:
+
+- **Restructuring the specification docs**: This section needs a better structure to provide easy access to the latest specification versions. Users need to understand how JSON Schema has evolved across different releases and the features introduced in each version. Restructuring this section makes it more useful for all the JSON Schema [personas](https://github.com/json-schema-org/community/issues/336#personas).
```
Comment (benjagm) [inline pages/blog/posts/GSOD-docs-project-2024.md:26]: We need to link the issue.

File: pages/blog/posts/GSOD-docs-project-2024.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+- **Restructuring the reference documentation**: We are creating a new version of the reference documents, using modern documentation principles, such as the [diataxis framework](https://diataxis.fr/), to improve the existing "Understanding JSON Schema" book. This will provide a more efficient onboarding experience and better content organization.
```
Comment (benjagm) [inline pages/blog/posts/GSOD-docs-project-2024.md:27]: We need to link the issue.

File: pages/blog/posts/GSOD-docs-project-2024.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
+---
+title: 'Update from the Google Season of Docs 2024 Project'
+date: '2024-09-03'
+tags:
+  - Google Season of Docs
+  - Documentation
+  - Community
+type: Documentation
+cover: /img/posts/2024/GSOD-cover/JSON-Schema-cover.png
```
Comment (benjagm) [inline pages/blog/posts/GSOD-docs-project-2024.md:9]: While the cover image is ok, I suggest you check what we did for other posts and search for a license-free image on Unsplash and use it.

File: pages/blog/posts/GSOD-docs-project-2024.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+- **Restructuring the reference documentation**: We are creating a new version of the reference documents, using modern documentation principles, such as the [diataxis framework](https://diataxis.fr/), to improve the existing "Understanding JSON Schema" book. This will provide a more efficient onboarding experience and better content organization.
+
+## The Update
+
+Since May, the team has been working hard to achieve these goals. We've conducted extensive research, mapped out [developer experience journeys](<(https://www.figma.com/board/TRjQUw33K93y8RlJMSRkJs/JSON-dev-journey?node-id=0-1&t=gYkdGtHk2sKrCQM0-0)>), implemented the [keywords page](https://json-schema.org/understanding-json-schema/keywords), and added on-page tables of content. We've also introduced [Next-Previous navigation](https://github.com/json-schema-org/website/pull/807) sections and restructured the [specification docs](https://github.com/json-schema-org/website/pull/823).
```
Comment (benjagm) [inline pages/blog/posts/GSOD-docs-project-2024.md:31]: Are the findings of the research in the figma project? If not we'll need to add the issue here as well.

**Review by benjagm — CHANGES_REQUESTED** [review]
Thanks a lot Blessing. I think this version looks much better and will be easier to follow to everyone. Well done!

I added few comments. Specifically, I think we need to add more details about @valeriahhdez 's work in progress for the Update section.

We are very close!

File: pages/blog/posts/GSOD-docs-project-2024.md (hunk 6 above)
Comment (kwennB) [reply pages/blog/posts/GSOD-docs-project-2024.md:31]: Yes the finding of the research are in the Figma project

**Review by benjagm — APPROVED** [review]

**Pull request merged**